src/map/index.js

```diff
@@ -116,10 +121,11 @@ const addZoomButtons = zoomFunction => {
     });
 };
 
-const buildZoom = (baseZoom, svgWidth, svgHeight) => {
+const buildZoom = (baseZoom, svgWidth, svgHeight, mapConfig) => {
+  const zoomScaler = mapConfig.zoomScalar || 4;
```

Comment:
Small thing but you've got `zoomScaler` and `zoomScalar`.

Reply:
Actually, where does `mapConfig.zoomScalar` get set?

Reply:
good catch - and these are getting set in the map's geojson. At the top level there's a `config` property which will let us change the projection and perform rotations / scale at map init